inspirehep/modules/migrator/tasks.py

```diff
@@ -221,7 +246,7 @@ def get_records_to_update_generator(citation_lookup):
                 },
                 "size": CHUNK_SIZE
             },
-            scroll=u'1m',
+            scroll=u'2m',
```

Comment:
You're right, it's not needed. Set it back to '1m'. (I played with it when I was working on synchronizing the migration process).